forge-gui/res/cardsfolder/upcoming/phelia_exuberant_shepherd.txt

```diff
@@ -0,0 +1,14 @@
+Name:Phelia, Exuberant Shepherd
+ManaCost:1 W
+Types:Legendary Creature Dog
+PT:2/2
+K:Flash
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Whenever CARDNAME attacks, exile up to one other target nonland permanent. At the beginning of the next end step, return that card to the battlefield under its owner's control. If it entered under your control, put a +1/+1 counter on NICKNAME.
+SVar:TrigExile:DB$ ChangeZone | ValidTgts$ Permanent.Other | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one other target permanent | Origin$ Battlefield | Destination$ Exile | RememberChanged$ True | SubAbility$ DelTrig
+SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigBounce | TriggerDescription$ Return exiled permanent to the battlefield. | RememberObjects$ RememberedLKI | SubAbility$ DBCleanup
+SVar:TrigBounce:DB$ ChangeZone | Origin$ Exile | Destination$ Battlefield | Defined$ DelayTriggerRememberedLKI | SubAbility$ DBPutCounter
+SVar:DBPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1 | ConditionDefined$ DelayTriggerRememberedLKI | ConditionPresent$ Creature
```

Comment:
wrong condition

Reply:
checking against LKI still seems wrong
imo best to imprint the zone change instead